pandas/__init__.py

```diff
@@ -188,6 +188,19 @@
     __git_version__ = v.get("full-revisionid")
     del get_versions, v
 
+import sys
```

Comment:
Does this break an API users potentially use. If so will we need a depr cycle?

Reply:
Kinda.
You can still access ``pandas.tests`` as before (if you're using a wheel without the tests submodule) e.g. if you wanted to use the extension dtypes tests, but you'll need to install ``pandas-tests`` to do so.

This is technically breaking, but the next release also happens to be 3.0, so I think we'd be able to get away with a breaking change here :)